lib/app/sequence_base.rb

```diff
@@ -813,6 +813,22 @@ def date_comparator_value(comparator, date)
           ''
         end
       end
+
+      def fetch_all_search_results(bundle)
```

Comment:
I think we discussed renaming this something like `fetch_all_bundle_pages`

Reply:
I changed it to fetch_all_bundled_resources